Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/ui/Question/QuestionAddServlet.java

```diff
@@ -0,0 +1,58 @@
+package ui.Question;
+
+import entity.Quiz;
+import entity.Question;
+import org.apache.log4j.Logger;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.*;
+
+
+public class QuestionAddServlet extends HttpServlet{
+    private static Logger log = Logger.getLogger(QuestionAddServlet.class.getName());
+
+
+
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+        this.process(request, response);
```

Comment:
Same logic for get and post is not good decision

Reply:
Why?
Somewhere read that if the servlet is simple and the use of get or post request does not matter - then you can use a separate method (including service ()) for stable workflow

Reply:
Because post for creating resources (processing forms) and get is for getting resources (web-pages or other data). They have different goal.